java/runtime-native/src/main/java/org/ray/spi/impl/StateStoreProxyImpl.java

```diff
@@ -80,6 +80,7 @@ public void checkConnected() throws Exception {
         return getAddressInfoHelper(nodeIpAddress);
       } catch (Exception e) {
         try {
+          RayLog.core.warn("StateStoreProxyImpl getAddressInfo", e);
```

Comment:
Let's make the error message more clear. E.g., 'Error occurred while getting address of ..., %d retries remaining.'

Reply:
Sure;)